controlsfx-samples/src/main/java/org/controlsfx/samples/HelloGlyphFont.java

```diff
@@ -138,6 +135,7 @@ public Node getPanel(final Stage stage) {
         	Color randomColor = new Color( Math.random(), Math.random(), Math.random(), 1);
         	Glyph graphic = Glyph.create( "FontAwesome|" + glyph.name()).sizeFactor(2).color(randomColor).useGradientEffect();
         	Button button = new Button(glyph.name(), graphic);
+            button.setMnemonicParsing(false);
```

Comment:
Indentation is off, can you fix that?

Reply:
And update the copyright header, replaced 2014 with 2025

Reply:
@Siedlerchr Thanks for the review! I’ll make the requested changes tomorrow morning.

Reply:
The misalignment was caused by a mix of tabs and spaces in the existing code. My change (button.setMnemonicParsing(false);) used spaces, which made it appear misaligned. I have now reformatted the for loop using spaces to ensure proper alignment on GitHub.

<img width="1343" alt="image" src="https://github.com/user-attachments/assets/12bdb8c1-b6b3-484d-aacc-13b7a8ee14b5" />